backend/postgresql/src/main/java/com/torodb/backend/postgresql/converters/PostgreSqlValueToCopyConverter.java

```diff
@@ -221,4 +236,11 @@ public Void visit(KvDeprecated value, StringBuilder arg) {
     arg.append(value.toString());
     return null;
   }
+
+  private StringBuilder appendJsonString(StringBuilder arg, String in) {
+    final String jsonb = in.replaceAll("\\\"", "\\\\\"");
```

Comment:
The code that write input for the COPY command should be optimized to prevent introducing overhead while writing to the backend. Using String.replaceAll here is too expensive due to the use of regular expression and could be substituted by a String.replace that would do the same at a lower cost.